src/locales/en-US/commands.json

```diff
@@ -419,60 +420,60 @@
     "following": "Following",
     "subcommands": {
       "track": {
-        "commandDescription": "Searches for a track on Deezer",
-        "commandUsage": "<track>",
-        "trackInfo": "Track information"
+        "commandDescription": "Searches for a song on Deezer.",
+        "commandUsage": "<song>",
+        "trackInfo": "Song information"
       },
       "album": {
-        "commandDescription": "Searches for an album on Deezer",
+        "commandDescription": "Searches for an album on Deezer.",
         "commandUsage": "<album> [--tracks]",
         "albumInfo": "Album information",
-        "albumTracks": "Album tracks"
+        "albumTracks": "Album songs"
       },
       "artist": {
-        "commandDescription": "Searches for an artist on Deezer",
+        "commandDescription": "Searches for an artist on Deezer.",
         "commandUsage": "<artist> [--albums | --related]",
         "artistInfo": "Artist information",
         "artistAlbums": "Artist albums",
         "artistRelated": "Related artists"
       },
       "playlist": {
-        "commandDescription": "Searches for a playlist on Deezer",
+        "commandDescription": "Searches for a playlist on Deezer.",
         "commandUsage": "<playlist> [--tracks]",
         "playlistInfo": "Playlist information",
-        "playlistTracks": "Playlist tracks"
+        "playlistTracks": "Playlist songs"
       },
       "podcast": {
-        "commandDescription": "Searches for a podcast on Deezer",
+        "commandDescription": "Searches for a podcast on Deezer.",
         "commandUsage": "<podcast> [--episodes]",
         "podcastInfo": "Podcast information",
         "podcastEpisodes": "Podcast episodes",
         "moreEpisodes": "...and {{episodes}} more episodes"
```

Comment:
I think this could use punctuation as well, but I'm not a 100% sure.

Reply:
Fix coming in next commit.